content/surveys/_index.en.md

```diff
@@ -7,33 +7,80 @@ weight: 15
 
 # Survey Templates
 
-We currently have three survey templates available in the [sitrep] package:
+The survey templates have been developed to match the data that would be collected using MSF Ethical Review Board pre approved surveys ([available here](https://fieldresearch.msf.org/handle/10144/618942)) on: ***retrospective mortality and access to care***, ***vaccination coverage*** and ***malnutrition***. 
```

Comment:
It's better to include more text within the link for accessibility:

`[MSF Ethical Review Board pre-approved surveys](https://fieldresearch.msf.org/handle/10144/618942)`